### PR TITLE
docs: add html field-label stories

### PR DESCRIPTION
**Pull request by josokinas**
## Purpose

After https://github.com/onfido/castor/pull/249 HTML (+ CSS) stories can now be added to each component.

## Approach

Adding stories to FieldLabel component.

Please note this excludes stories that include Validation component.

## Testing

On Storybook.

## Risks

N/A

### CODE REVIEW

File: docs/helpers/html.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ export const html = (
     `<${name}`,
     Object.entries(props).map(toAttribute).join('\n'),
     '>',
-    children,
+    Array.isArray(children) ? children.join('') : children,
```
Comment (josokinas) [inline docs/helpers/html.ts:15]: I'm not sure about this, what do you think @rabelloo maybe better just each time to join on each use?

**Comment by rabelloo** [comment]
The snippet contains the output code rather than the Castor helpers, as it should tbf.

image

Not that this is a problem, but ideally it'd be `color('token-foo')` instead.

Any ideas?

File: packages/core/src/components/field-label/field-label.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  Field({
+    children: [
+      FieldLabel({
+        ...props,
+        children: label,
+        for: id,
+      }),
+      Input({ id }),
+    ],
+  });
```
Comment (rabelloo) [inline packages/core/src/components/field-label/field-label.stories.tsx:78]: Remind me again why we do `input` outside of `label` with `[id]` and `[for]`?

Comment (rabelloo) [reply packages/core/src/components/field-label/field-label.stories.tsx:78]: i.e. 
image

**Review by rabelloo — APPROVED** [review]

File: packages/core/src/components/field-label/field-label.stories.tsx (hunk 2 above)
Comment (josokinas) [reply packages/core/src/components/field-label/field-label.stories.tsx:78]: This was to demonstrate different ways of using field label with an input.

FieldLabel has examples of using `[for]`, where Input examples show as using nesting within `label`.

However, I have noticed that on HTML + CSS I used `[for]` everywhere... Will change.

**Comment by josokinas** [comment]
> Any ideas?

I see the same output is for both storeis, it renders the output of a helper function.

TBH I'm not sure which is more correct, both are valid. And this might even be better, as it's an example for whether you use helper or you don't.

**Comment by rabelloo** [comment]
> TBH I'm not sure which is more correct, both are valid. And this might even be better, as it's an example for whether you use helper or you don't.

That's a good point, I'm convinced.

**Pull request merged**